src/components/WhoWeAre/Team.tsx

```diff
@@ -14,8 +14,8 @@ const Team = (props: any) => {
         width="100%"
       />
       <div className="text-center">
-        <h3>{name}</h3>
-        <h4>({pronoun})</h4>
+        <TeamName>{name}</TeamName>
```

Comment:
It's not semantic to go from h3 to h5 without an h4 in between. The hierarchy of header sizes should always be in order.

Reply:
From what the original production of this section, and how Alison has H4 set as italics. These would be the Name and pronouns from the Design File.

Reply:
![example](https://user-images.githubusercontent.com/15990675/110022260-4fa35980-7cf1-11eb-9e29-72445194f6ac.png)

This section of code will be updated soon in DES-6.